docs/roadmap.md

```diff
@@ -0,0 +1,60 @@
+# `kind` RoadMap 🗺️
+
+New year, new roadmap 🎉  
+This document outlines some goals, non-goals, and future aspirations for kind
+as a project.
+
+High level goals for kind include:
+
+- Reach [1.0] status / 
+stability by the end of Q1 2019 / the Kubernetes 1.14 release cycle
+- Support multi-node clusters - [#117]
+- Support offline / air-gapped clusters
+  - pre-loaded / offline CNI - [#200]
+- Improve Windows support
+  - add Windows binaries to releases - [#155]
+  - improve instructions for KUBECONFIG in particular
+- Improve documentation
+  - expand examples of using kind
+  - cover known issues, debugging, work-arounds, etc.
+  - create a GitHub-pages landing page with docs
+- Improving logging and debuggability
+  - enable audit-logging
+- Support non-amd64 architectures (namely ARM) - [#166]
+- Automated publishing of Kubernetes release based kind "node" images - [#197]
+
+**Non**-Goals include:
+
+- Supporting every possible Kubernetes configuration
+  - In order to best support offline / hermetic clusters, we will likely not
+  offer many options for CNI etc. out of the box. We may revisit this later.
+- Being "production workload ready" - kind is meant to be used:
+  - for testing Kubernetes itself
+  - for testing against Kubernetes (EG in CI on Travis, Circle, etc.)
+  - for "local" clusters on developer machines
+  - NOT to host workloads serving user traffic etc.
+- Replacing [Phippy] 🦒 -- kind isn't trying to replace all the things
+and Phippy is awesome ❤️
+
+Longer Term goals include:
+- Enabling a suitable local storage provider for testing applications that need
+persistent storage
+- Improving "kind as a library"
+
+Misc:
+ - setup a regular Zoom meeting for the project
+ - acheive certified Kubernetes conformance
```

Comment:
achieve :-)

Reply:
thanks! fixed and ran misspell over it to check for anything else